packages/@aws-cdk/aws-cloudformation/test/test.pipeline-actions.ts

```diff
@@ -341,7 +341,7 @@ class StageDouble implements cpapi.IStage {
     const stageParent = new cdk.Construct(pipeline, this.stageName);
     for (const action of actions) {
       const actionParent = new cdk.Construct(stageParent, action.actionName);
-      (action as any)._attachActionToPipeline(this, actionParent);
+      (action as any)._actionAttachedToPipeline(this, pipeline.role, actionParent);
```

Comment:
It's because I wanted to have a different name for the method called on Action then the one Stage calls on the Pipeline itself (which is called `_attachActionToPipeline`).

The past tense is to mimic callback-based APIs, like `onButtonPressed`, that you sometimes see in GUI toolkits, for example. I'm very open to suggestions on a better name, of course :).

Reply:
`_bind`?

Reply:
... is already taken :)

Reply:
Well, technically it's `bind` that's already taken, but I'm not sure having both `bind` and `_bind` in the same class is the best idea :)